README.md

```diff
@@ -1,13 +1,14 @@
 # IsraelTechWriters
+
 This repo aim to gather all Israelis tech writer, feel free to open Pull Request and add your self❤️
 Attencion, please add ": " and than with comma separator the topics covered on the blog, and then ". Writing language: " and than with comma separator en / he.
-For example: 
+For example:
 "Front end, computer science. Writing language: en."
 
-
 * [Pull Request Community Blog](https://pullrequestcommunityisrael.medium.com/): open source. Writing language: en.
 * [Michal Porag](https://michal-porag.medium.com/): Front end, computer science. Writing language: en.
 * [Elad Shechter](https://elad.medium.com/): CSS. Writing language: en.
 * [David Meir-Levy](https://www.davidlevy.co.il): Front end. Writing language: en.
 * [Uri Shaked](https://urish.org): IoT, Electronics, Frontend and more. English.
 * [camelCase](https://www.camelCase.blog): Software Engineering. Writing language: he.
+* [Nitzan Ohana](https://blog.nitzano.com): Full stack, open source and more. Writing language: en.
```

Comment:
Please remove "and more" or change to what it is about.

Reply:
@MichalPorag: done